src/main/java/eu/enexa/module/FileBasedModuleManager.java

```diff
@@ -46,6 +46,12 @@ public FileBasedModuleManager() throws IOException {
                 LOGGER.info(enexaModuleDirectory.getAbsolutePath() + " exist");
                 addFileOrDirectory(enexaModuleDirectory);
                 LOGGER.info("modules size is : "+ modules.size());
+                for (Map.Entry<String, ModuleModel> entry : modules.entrySet()) {
+                    String key = entry.getKey();
+                    ModuleModel module = entry.getValue();
+                    System.out.println("Key: " + key + ", Value: " + module.toString());
+                }
+
```

Comment:
If this information is needed for debugging, it should be logged using a Logger.

If it was there only temporarily, it should be removed.

Reply:
yes it was just for debugging,  deleted